b-real-time-htb.js

```diff
@@ -233,24 +255,22 @@ function BRealTimeHtb(configs) {
 
         /* ---------- Proces adResponse and extract the bids into the bids array ------------*/
 
-        var bids = adResponse;
+        var bids = adResponse.tags;
 
         /* --------------------------------------------------------------------------------- */
-
         for (var i = 0; i < bids.length; i++) {
 
-            var curReturnParcel;
-
+            var curReturnParcel, bid = bids[i];
+            
             for (var j = unusedReturnParcels.length - 1; j >= 0; j--) {
-
+                var unusedReturnParcel = unusedReturnParcels[j];
                 /**
                  * This section maps internal returnParcels and demand returned from the bid request.
                  * In order to match them correctly, they must be matched via some criteria. This
                  * is usually some sort of placements or inventory codes. Please replace the someCriteria
                  * key to a key that represents the placement in the configuration and in the bid responses.
                  */
-
-                if (unusedReturnParcels[j].someCriteria === bids[i].someCriteria) { // change this
+                if (unusedReturnParcel.xSlotRef.uuid === bid.uuid) { // change this
```

Comment:
As I understand it, the UUID here is what is passed from the endpoint in a 'live' environment, correct? If so, the uuid passed on line 172 should be within the payload, our endpoint should pass it along.

Unfortunately no, a placementID can be used with multiple slots in some cases.